server.js

```diff
@@ -7,6 +7,23 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
```

Comment:
I'm sure you know, but just a reminder to be consistent with the use of ";", even in the imports.